packages/agreement-process/src/routers/AgreementRouter.ts

```diff
@@ -194,12 +195,18 @@ const agreementRouter = (
     }
   );
 
-  agreementRouter.post(
-    "/agreements/:agreementId/archive",
-    async (_req, res) => {
-      res.status(501).send();
+  agreementRouter.post("/agreements/:agreementId/archive", async (req, res) => {
```

Comment:
Missing `authorizationMiddleware([ADMIN_ROLE])`?

Reply:
Thank you! 🙏🏻